src/doc/en/developer/trac.rst

```diff
@@ -280,7 +280,7 @@ To produce the warning message, use code like the following:
         TICKET_NUM)
```

Comment:
Best to revert the changes to this file?

Reply:
Agreed.